packages/remix-server-runtime/sessions/cookieStorage.ts

```diff
@@ -37,7 +37,12 @@ export function createCookieSessionStorage({
       );
     },
     async commitSession(session, options) {
-      return cookie.serialize(session.data, options);
+      const serializedCookie = await cookie.serialize(session.data, options);
+      const byteLength = Buffer.byteLength(serializedCookie);
```

Comment:
We can't use node primitives like `Buffer` in `@remix-run/server-runtime`, because this package is supposed to be platform-agnostic.  Unless we care about multi-byte characters here, it should be acceptable to just check `serializedCookie.length` instead.

Reply:
Also, please reuse existing code conventions and use `let` instead of `const` for local variables.

Reply:
Resolved both of these. Thanks for pointing out @mjackson .

On the let vs. const - this convention is totally opaque to newcomers and seems like it would be a hassle for maintainers to repeat. Perhaps an [eslint plugin](https://www.npmjs.com/package/eslint-plugin-prefer-let) would be an easier way to enforce things like this.